app/api.py

```diff
@@ -209,6 +211,87 @@ async def sentiment(text: str):
     return {"result": vader_score(text)}
 
 
+@API.get("/graphs/tech-stack-by-role")
+async def tech_stack_graph():
+    """ PLACEHOLDER - NOT WORKING """
+    mentors_df = pd.DataFrame(API.db.read("Mentors"))[["tech_stack", "name"]]
+    mentees_df = pd.DataFrame(API.db.read("Mentees"))[["tech_stack", "name"]]
+    mentors_df["user_role"] = "Mentor"
+    mentees_df["user_role"] = "Mentee"
+    df = pd.concat([mentees_df, mentors_df], axis=0).reset_index(drop=True)
+    return json.loads(tech_stack_by_role(df).to_json())
+
+
+@API.get("/responses_analysis")
+async def responses_analysis():
+    """Returns relevant topics for analysis usage
+
+   calls the nlp_analysis function from the analysis.py file.
+
+    Returns:
+        (dict)relevant topics from the responses collection
+    """
+    return nlp_analysis([obj["text"] for obj in API.db.read("Responses")])
+
+
+@API.post("/read/feedback")
+async def read_feedback(query: FeedbackOptions):
+    """Read records in the feedback collection.
+    Reads new document within Feedback using the ticket_id  or mentor_id parameter to
+    populate its fields.
+    """
+    return {"result": API.db.read("Feedback", query.dict())}
+
+
+@API.post("/create/feedback")
+async def create_feedback(data: Feedback):
+    """Create records in the feedback collection.
+
+        Creates new document within Feedback using the Feedback schema to
+        populate its fields. Creates ticket_id and vader_score automatically on submission
```

Comment:
This docstring suggests that a ticket_id will be created automatically. I'm pretty sure this is incorrect.

Reply:
Correct. While not the most elegant thing I've written, I believe this edit dispels the misinformation: """"Create records in the feedback collection.
        Creates new document within Feedback using the Feedback schema to
        populate its fields, except for ticket_id, which comes from FE."